kuma/wiki/content.py

```diff
@@ -183,6 +183,13 @@ def clean_content(content):
     else:
         allowed_iframe_patterns = settings.ALLOWED_IFRAME_PATTERNS
         filtered = parsed.filterIframeHosts(allowed_iframe_patterns)
+
+    # NOTE(djf): if you ever want to compare HTML renderings of a document
+    # (before and after a KumaScript or macro change, for example) you
+    # may need to add alphabetical_attributes=True to the serialize call
+    # below so that attributes are output in a deterministic order. This was
+    # a particular problem for <iframe> tags, presumably because of the
+    # iframe filtering above.
```

Comment:
I think adding ``alphabetical_attributes=True`` would be a good stand-alone change, then we could get rid of this comment.

Reply:
It seems like it might only be iframes that have their attribute ordering changed? If so, then forcing alphabetical might drive writers nuts when they look at their diffs?